src/test/java/uk/gov/hmcts/reform/waworkflowapi/controllers/startworkflow/CreateTaskControllerTest.java

```diff
@@ -0,0 +1,117 @@
+package uk.gov.hmcts.reform.waworkflowapi.controllers.startworkflow;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import uk.gov.hmcts.reform.waworkflowapi.clients.model.DmnValue;
+import uk.gov.hmcts.reform.waworkflowapi.clients.model.EvaluateDmnRequest;
+import uk.gov.hmcts.reform.waworkflowapi.clients.model.EvaluateDmnResponse;
+import uk.gov.hmcts.reform.waworkflowapi.clients.model.SendMessageRequest;
+import uk.gov.hmcts.reform.waworkflowapi.clients.service.EvaluateDmnService;
+import uk.gov.hmcts.reform.waworkflowapi.clients.service.SendMessageService;
+import uk.gov.hmcts.reform.waworkflowapi.common.TaskToCreate;
+import uk.gov.hmcts.reform.waworkflowapi.duedate.DueDateService;
+
+@ExtendWith(MockitoExtension.class)
+class CreateTaskControllerTest {
+
+    private static final String CREATE_TASK_MESSAGE = "createTaskMessage";
+
+    @Mock
+    private EvaluateDmnService evaluateDmnService;
+
+    @Mock
+    private SendMessageService sendMessageService;
+
+    @Mock
+    private SendMessageRequest sendMessageRequest;
+
+    @Mock
+    private DueDateService dueDateService;
+
+    private CreateTaskController createTaskController;
+
+    @BeforeEach
+    void setUp() {
+        createTaskController = new CreateTaskController(
+            evaluateDmnService,
+            sendMessageService,
+            dueDateService
+        );
+    }
+
+    @Test
+    void evaluateDmn() {
+        final String key = "Key";
+        final DmnValue<?> dmnValue = mock(DmnValue.class);
+        final List<Map<String, DmnValue<?>>> evaluateDmnResponse = new ArrayList<Map<String, DmnValue<?>>>();
+        final Map<String, DmnValue<?>> map = new HashMap<String, DmnValue<?>>();
+        map.put(key, dmnValue);
+        evaluateDmnResponse.add(map);
+        EvaluateDmnRequest evaluateDmnRequest = mock(EvaluateDmnRequest.class);
+
+        when(evaluateDmnService.evaluateDmn(evaluateDmnRequest, key)).thenReturn(evaluateDmnResponse);
+        ResponseEntity<EvaluateDmnResponse> response = createTaskController.evaluateDmn(evaluateDmnRequest, key);
+
+        assertNotNull(response);
+        assertEquals(HttpStatus.OK, response.getStatusCode());
+        assertThat(response.getBody(), instanceOf(EvaluateDmnResponse.class));
+        assertNotNull(response.getBody());
```

Comment:
Duplicate assertion, line 75 does validate to a value. Or you can move line 76 above 75.